src/Symfony/Bridge/Twig/DataCollector/TwigDataCollector.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\DataCollector;
+
+use Symfony\Component\HttpKernel\DataCollector\DataCollector;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * TwigDataCollector.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+class TwigDataCollector extends DataCollector
+{
+    private $profile;
+    private $computed;
+
+    public function __construct(\Twig_Profiler_Profile $profile)
+    {
+        $this->profile = $profile;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function collect(Request $request, Response $response, \Exception $exception = null)
```

Comment:
shouldn't this use ``lateCollect`` to take into account templates rendered later (in a streamed response for instance) ?